.github/workflows/main.yml

```diff
@@ -23,9 +23,10 @@ jobs:
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
 
-    - name: Execute bootstrap.sh
+    - name: Install basic packages
       run: |
-        sudo ./bootstrap.sh
+        sudo apt-get update -y
+        sudo apt-get install -y python3-pip python3-setuptools libsystemd-dev
```

Comment:
Which process is dependent on those packages?

Reply:
> Which process is dependent on those packages?

Libraries from tests/requirements-test.txt require those packages. 
Their installation wouldnt work without those packages. I checked. 
I think that pystemd (which is used i n the tests) is one such library.

Reply:
Please add a comment that those are required for testing and maybe place this task near the testing task.

Reply:
I made the 'name' section more informative. I dont this we should move this whole section down near tests execution, it should remain here along with other installations.